episodes/introduction.md

```diff
@@ -1,102 +1,95 @@
 ---
-title: "Using Markdown"
-teaching: 10
-exercises: 2
+title: "Welcome"
```

Comment:
think this episode as a whole will probably need more text about the reading each week and discussion sessions to set the stage for how trainer training will run.